### DISCUSSION TITLE
Foundry installing issue

### DISCUSSION THREAD

**Discussion by doichantran** [Cyfrin/foundry-full-course-cu#2184] [General]
Every time I put this in VS Code terminal
```
curl -L https://foundry.paradigm.xyz | bash
```
I always get this error
```
PS C:\Users\PhiliaDX-Admin> curl -L https://foundry.paradigm.xyz | bash
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   167  100   167    0     0   1128      0 --:--:-- --:--:-- --:--:--  1136
100  2189  100  2189    0     0   6119      0 --:--:-- --:--:-- --:--:-- 45604
: invalid option nameipefail
```
Could someone help me

**Comment by EngrPips** [comment]
Hello @doichantran , have you checked the official `Foundry` GitHub to see if there is a similar issue and maybe see how it was resolved? I have no idea what is wrong.

**Comment by humbletolearn** [comment]
Hello @doichantran , you should run linux environment first by  "wsl"
Then, run   curl -L https://foundry.paradigm.xyz | bash
